opentech/apply/funds/views.py

```diff
@@ -345,6 +347,35 @@ def form_valid(self, form):
         return super().form_valid(form)
 
 
+@method_decorator(staff_required, name='dispatch')
+class CreateProjectView(DelegatedViewMixin, CreateView):
+    context_name = 'project_form'
+    form_class = CreateProjectForm
+    model = Project
+
+    def form_valid(self, form):
+        submission_id = form.cleaned_data['submission']
+        try:
+            submission = ApplicationSubmission.objects.get(pk=submission_id)
+        except ApplicationSubmission.DoesNotExist:
+            return self.form_invalid(form)
+
+        project, created = Project.create_from_submission(submission)
+
+        if not created:
+            return self.form_invalid(form)
+
+        messenger(
+            MESSAGES.CREATED_PROJECT,
+            request=self.request,
+            user=self.request.user,
+            submission=submission,
+            project=project,
+        )
+
+        return redirect('apply:submissions:detail', submission.id)
```

Comment:
The following would work as well i think. 
```
return redirect(self.get_success_url())
```

Reply:
Fixed in 6c7dd74b